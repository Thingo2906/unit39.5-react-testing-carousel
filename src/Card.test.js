import React from "react";
import { render } from "@testing-library/react";
import Card from './Card';
import image1 from "./image1.jpg";
// smoke test
test('it render without crashing', () => {
    render(<Card caption ="Hawaii island" src = {image1} currNum = {1} totalNum = {3}/>)
});
//snapshot test
test('it match snapshot', () => {
    const { asFragment } = render(
      <Card caption="Hawaii island" src={image1} currNum={1} totalNum={3} />
    );
    expect(asFragment()).toMatchSnapshot();
})