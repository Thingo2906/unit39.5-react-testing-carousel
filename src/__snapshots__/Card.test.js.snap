// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Hawaii island
    </h4>
    <img
      alt="Hawaii island"
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
